requirements.txt

```diff
@@ -1,23 +0,0 @@
-astroid==1.6.5
```

Comment:
I think you need to link actual requirements from here. Try deploying this branch.

Reply:
Good, I'll try it.

Reply:
@webknjaz https://github.com/PyCQA/pylint-django/blob/7a7788795c488bb145089b73dc85bd5bda39e31a/pylint_django/transforms/__init__.py#L5 `astroid` in `pylint-django`

Reply:
It's about having another file in root